core/src/test/scala/org/apache/spark/storage/MemoryStoreSuite.scala

```diff
@@ -291,11 +291,11 @@ class MemoryStoreSuite
     blockInfoManager.removeBlock("b3")
     putIteratorAsBytes("b3", smallIterator, ClassTag.Any)
 
-    // Unroll huge block with not enough space. This should fail and kick out b2 in the process.
+    // Unroll huge block with not enough space. This should fail.
```

Comment:
Seems OK to me. Maybe @JoshRosen would like to comment. If the allocation is known to fail because it can't possibly allocate that much memory, I agree that there's no need to evict blocks.